deploy/archive-sync-ols.yaml

```diff
@@ -0,0 +1,291 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: archive-sync-ols
+objects:
+
+- kind: HorizontalPodAutoscaler
+  apiVersion: autoscaling/v1
+  metadata:
+    labels:
+      app: ccx-data-pipeline
+    name: archive-sync-ols
+  spec:
+    minReplicas: ${{MIN_REPLICAS}}
+    maxReplicas: ${{MAX_REPLICAS}}
+    scaleTargetRef:
+      apiVersion: apps/v1
+      kind: Deployment
+      name: archive-sync-ols-instance
+    targetCPUUtilizationPercentage: 50
+
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: archive-sync-ols
+  spec:
+    envName: ${ENV_NAME}
+    objectStore:
+      - ${TARGET_S3_BUCKET}
+#    testing:
```

Comment:
is this needed? leftover?

Reply:
No, I left intentionally. I want to make clear that we are not testing with IQE right now, but might be the best path forward.